src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -191,73 +184,35 @@ private void testCreateInstructorAction() throws Exception {
         coursesPage.loadInstructorHomeTab();
         instructorHomePage = AppPage.getNewPageInstance(browser, InstructorHomePage.class);
-        instructorHomePage.clickFeedbackSessionViewResultsLink(demoCourseId, "Second team feedback session");
-        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackResultsPage.html");
-
-        ______TS("new instructor can edit feedbackSession of sample course");
-        instructorHomePage.loadInstructorHomeTab();
-        InstructorFeedbackEditPage feedbackEditPage =
-                instructorHomePage.clickFeedbackSessionEditLink(demoCourseId, "Second team feedback session");
-
-        feedbackEditPage.clickEditSessionButton();
-
-        FeedbackSessionAttributes feedbackSession =
-                getFeedbackSessionWithRetry(demoCourseId, "Second team feedback session");
-        feedbackEditPage.editFeedbackSession(feedbackSession.getStartTime(),
-                                             feedbackSession.getEndTime(),
-                                             new Text("updated instructions"),
-                                             feedbackSession.getGracePeriod());
-        feedbackEditPage.reloadPage();
-        instructorHomePage.verifyHtmlMainContentWithReloadRetry("/newlyJoinedInstructorFeedbackSessionSuccessEdited.html");
-
-        ______TS("new instructor can click submit button of sample feedbackSession");
-        instructorHomePage.loadInstructorHomeTab();
-        FeedbackSubmitPage fbsp = instructorHomePage.clickFeedbackSessionSubmitLink(demoCourseId,
-                                                                                    "Second team feedback session");
-        fbsp.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackSubmissionEditPage.html");
-
-        ______TS("new instructor can send reminder of sample course");
-        instructorHomePage.loadInstructorHomeTab();
-        instructorHomePage.clickFeedbackSessionRemindLink(demoCourseId,
-                                                          "Second team feedback session");
-        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackSessionRemind.html");
-
-        ______TS("new instructor can unpublish feedbackSession of sample course");
-        instructorHomePage.loadInstructorHomeTab();
-        instructorHomePage.clickFeedbackSessionUnpublishLink(demoCourseId, "Second team feedback session");
-        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackSessionUnpublished.html");
-
-        ______TS("new instructor can publish feedbackSession of sample course");
-        instructorHomePage.loadInstructorHomeTab();
-        instructorHomePage.clickFeedbackSessionPublishLink(demoCourseId, "Second team feedback session");
-        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackSessionPublished.html");
-
-        feedbacksPage.logout();
```

Comment:
It's better now, but can be even more concise. 
This whole block
```java
        ______TS("new instructor can view result of First team feedback session of sample course");
        AppUrl url = createUrl(Const.ActionURIs.INSTRUCTOR_COURSES_PAGE)
                .withUserId(TestProperties.TEST_INSTRUCTOR_ACCOUNT);
        InstructorCoursesPage coursesPage = AppPage.getNewPageInstance(browser, url, InstructorCoursesPage.class);
        coursesPage.waitForAjaxLoadCoursesSuccess();
        coursesPage.loadInstructorHomeTab();
        instructorHomePage = AppPage.getNewPageInstance(browser, InstructorHomePage.class);
        InstructorFeedbackResultsPage resultsPage =
                instructorHomePage.clickFeedbackSessionViewResultsLink(demoCourseId, "First team feedback session");
        resultsPage.expandPanels();
        resultsPage.verifyHtmlMainContent("/newlyJoinedInstructorFirstFeedbackSessionResultsPage.html");

        ______TS("new instructor can view result of Second team feedback session of sample course");
        coursesPage.loadInstructorHomeTab();
        instructorHomePage = AppPage.getNewPageInstance(browser, InstructorHomePage.class);
        resultsPage =
                instructorHomePage.clickFeedbackSessionViewResultsLink(demoCourseId, "Second team feedback session");
        resultsPage.expandPanels();
        resultsPage.verifyHtmlMainContent("/newlyJoinedInstructorSecondFeedbackSessionResultsPage.html");

        ______TS("new instructor can view result of Third team feedback session of sample course");
        coursesPage.loadInstructorHomeTab();
        instructorHomePage = AppPage.getNewPageInstance(browser, InstructorHomePage.class);
        resultsPage = instructorHomePage.clickFeedbackSessionViewResultsLink(
                demoCourseId, "Session with different question types");
        resultsPage.expandPanels();
        resultsPage.verifyHtmlMainContent("/newlyJoinedInstructorThirdFeedbackSessionResultsPage.html");
```
can be collapsed to
```java
        ______TS("new instructor can view feedback sessions of sample course");
        AppUrl url = createUrl(Const.ActionURIs.INSTRUCTOR_COURSES_PAGE)
                .withUserId(TestProperties.TEST_INSTRUCTOR_ACCOUNT);

        verifyCanViewSesssion(demoCourseId, "First team feedback session",
                "/newlyJoinedInstructorFirstFeedbackSessionResultsPage.html");        
        verifyCanViewSesssion(demoCourseId, "Second team feedback session",
                "/newlyJoinedInstructorSecondFeedbackSessionResultsPage.html");  
        verifyCanViewSesssion(demoCourseId, "Third team feedback session",
                "/newlyJoinedInstructorThirdFeedbackSessionResultsPage.html");  
```

What do you think?